platform/resource-bundle-locator/src/test/java/ddf/platform/resource/bundle/locator/ResourceBundleLocatorImplTest.java

```diff
@@ -0,0 +1,72 @@
+/**
+ * Copyright (c) Codice Foundation
+ *
+ * <p>This is free software: you can redistribute it and/or modify it under the terms of the GNU
+ * Lesser General Public License as published by the Free Software Foundation, either version 3 of
+ * the License, or any later version.
+ *
+ * <p>This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+ * without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Lesser General Public License for more details. A copy of the GNU Lesser General Public
+ * License is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+package ddf.platform.resource.bundle.locator;
+
+import static org.hamcrest.core.Is.is;
+import static org.junit.Assert.assertThat;
+
+import java.util.Locale;
+import java.util.MissingResourceException;
+import java.util.ResourceBundle;
+import org.junit.Before;
+import org.junit.Test;
+
+public class ResourceBundleLocatorImplTest {
+
+  private ResourceBundleLocatorImpl resourceBundleLocator;
+
+  private static final String TEST_BASE_NAME = "TestResourceBundle";
+
+  @Before
+  public void setup() {
+    this.resourceBundleLocator = new ResourceBundleLocatorImpl();
+    resourceBundleLocator.setResourceBundleBaseDir("src/test/resources/");
```

Comment:
❓ Have you tested running this test case from both IntelliJ and the command line?

Reply:
I've run these tests on the command line, through IntelliJ, and the CI. Since this module is creating file objects is there a better way to do this?

Reply:
This was just to confirm that accessing the resources from disk would work well in all cases. Make sure to also run on both Linux (or Mac) and Windows since you do have path-specific code

Reply:
I'm going to have someone with a Windows machine test this and I'll verify that the tests worked as intended.

Reply:
Just verified with @willwill96 that these unit tests pass on a windows machine.